open-api/rest-catalog-open-api.yaml

```diff
@@ -114,7 +117,13 @@ paths:
                 },
                 "defaults": {
                   "clients": "4"
-                }
+                },
+                "endpoints": [ "GET /v1/{prefix}/namespaces/{namespace}",
+                               "GET /v1/{prefix}/namespaces",
```

Comment:
Formatting is a little awkward here.  Maybe start the list entries on a new line?

Reply:
done